docs/index.adoc

```diff
@@ -3,19 +3,38 @@
 
 The international Software Architecture Qualification Board (link:https://isaqb.org[iSAQB]) defines curricula on several levels for software architects.
 
-You can find the Template Level Curriculum (short: TLC) for the foundation level at <<foundation-curriculum.adoc#>>
-
-The terminology used in the iSAQB curricula can be found as a (freely available) ebook, published on https://leanpub.com/isaqbglossary/read[Leanpub]
+The terminology used in the iSAQB curricula can be found as a (freely available) ebook, published on https://leanpub.com/isaqbglossary/read[Leanpub].
 
 == Production version
 
-The current productive version (aka: relevant for examinations) is version 4.1 (EN) and 4.2 (DE), both of which are currently only available via the official https://isaqb.org[iSAQB website].
+The current productive version (i.e. relevant for examinations) is version 4.1 (EN) and 4.2 (DE), both of which are currently only available via the official https://isaqb.org[iSAQB website].
```

Comment:
Das stimmt auch nicht. Das ist nur für den Foundation level so, kann komplett raus. Von diesem Template gibt es ja keine Version, es soll ja für andere Advanced-Level-Schulungen nur die Vorlage sein. Diese wiederum existieren dann in einer Version x.y